### ISSUE TITLE
Add support for Variable declarations (let)

### ISSUE THREAD

**Issue by gcanti** [gcanti/babel-plugin-tcomb#70]
Example

```js
let bar: bool = foo;

bar = true;
```

compiles to

```js
let bar = _assert(foo, _t.Boolean, "bar");

bar = _assert(true, _t.Boolean, "bar");
```

@christophehurpeau let me know if you want to work on this. From my part I'd love to help you out writing a test suite which I'll post later here or in a apposite branch

**Comment by christophehurpeau** [comment]
`bar = true;` should not compile to `bar = _assert(true, _t.Boolean, "bar");` because we know that `true` is a boolean, in my opinion.

**Comment by gcanti** [comment]
Yeah, but seems just an optimization which btw implies a bit of (albeit basic) static analysis, I think it doesn't worth it to complicate the plugin code for this. In the end of the day `assert`s will be discarded in production builds

**Comment by christophehurpeau** [comment]
see #111

**Comment by ArmorDarks** [comment]
Hi!

It's slightly unclear — plugin still does not support lets?
